CHANGELOG.md

```diff
@@ -218,12 +262,17 @@ See the [release announcement](TBD) for details.
 - [Various fixes to Variant and Reference pointers](https://github.com/godotengine/godot/pull/43049).
 - [Optimized octree and fixed a leak](https://github.com/godotengine/godot/pull/41123).
 - [Fix crash when opening a ZIP data pack](https://github.com/godotengine/godot/pull/42196).
+- [`OS.delay_msec()` and `OS.delay_usec()` with a negative value no longer freeze the engine and return an error message instead](https://github.com/godotengine/godot/pull/46194).
 - Various fixes to CSG mesh generation. There should be less instances of holes being created in CSG meshes.
 
 #### Editor
 
 - [Fix exporting if the temporary export directory doesn't exist yet](https://github.com/godotengine/godot/pull/45561).
 - [Objects can no longer be added to the Project Settings since they can't be serialized correctly](https://github.com/godotengine/godot/pull/42616).
+- [Fix hidden nodes being selectable in the 2D editor in specific cases](https://github.com/godotengine/godot/pull/46261).
+- [The default float step for inspector properties is now defined with a `double`, fixing issues with ProgressBar precision](https://github.com/godotengine/godot/pull/47408).
```

Comment:
Should minor bugfixes really be included in the changelog?

Reply:
Yeah I wouldn't necessarily bother with such minor issues that don't really impact users' workflow.

What's important to include in the changelog is:
- Cool stuff that will make users' life easier, so they need a way to know that this stuff was added.
- Changes/bugfixes that will solve long standing issues that would have been a problem in their workflow.
- Changes/bugfixes that will impact their workflow (or project behavior) and which they need to be aware of to see if/how they want to take this into account (and particularly compat breaking changes).

Reply:
That being said most of the changes added here do fit these criteria so IMO it's OK. Better have a few tiny fixes mentioned than missing big changes. (But otherwise yeah I think this entry specifically could be removed for consistency with many other tiny edits which are not featured.)